### ISSUE TITLE
Option for generating lenses

### ISSUE THREAD

**Issue by timjb** [k-bx/protocol-buffers#11]
It would be great if `hprotoc` had an option `--lenses` for generating lenses for all data types.

This requires prepending an underscore to all field names and inserting a call to `makeLenses` after each data type declaration:

```haskell
data BlobProto = BlobProto { _num :: ..., _channels :: ... } deriving (...)

makeLenses ''BlobProto
```

**Comment by k-bx** [comment]
I wouldn't mind this at all. @timjb care to try out and make a PR?

**Comment by timjb** [comment]
I've opened a pull request: https://github.com/k-bx/protocol-buffers/pull/12

**Comment by k-bx** [comment]
Released as 2.1.7 version
